### DISCUSSION TITLE
Add support for right-to-left languages

### DISCUSSION THREAD

**Discussion by pookmaster21** [d99kris/nchat#260] [Ideas]
**Description**:
chats that are in langueges that are right to left(hebrew, arabic etc.) show as left to right.

**How to reproduce it**:
any chat with a right to left languege(title or chat itself)

**Environment**:
- Version: v4.86
- OS / distro: arch linux

**Comment by d99kris** [comment]
Thanks for highlighting this issue. I suspect it would be quite a bit of work to support this, so I've decided to leave it out of scope for now, and converted this issue to a feature request.

If any developer, ideally familiar with RTL/right-to-left text in ncurses-based TUI applications, would like to help out with this, feel free to reach out.
